sites/docs/src/content/docs/checklists/pipeline_release.md

```diff
@@ -64,8 +64,9 @@ The last step is to bump up the pipeline version number in the development branc
   - [ ] Use the `nf-core pipelines bump-version` command to make the changes, eg: navigate to the pipeline directory and run `nf-core pipelines bump-version 1.1.0dev`
 - [ ] Update the `CHANGELOG.md` to include a new section for this new version
 - [ ] [Open a Pull Request (PR)](https://help.github.com/en/articles/creating-a-pull-request) with these changes from your fork to the `dev` branch on the nf-core repository.
-- [ ] (first release only) After the first release of the pipeline you will need to add the DOI manually into the main `README.md` for the pipeline:
+- [ ] (**First release only**) After the first release of the pipeline you will need to add the DOI manually into the main `README.md` for the pipeline:
   - [ ] Search for your pipeline on Zenodo and find the DOI that allows you to _"Cite all versions"_ of the pipeline.
   - [ ] Ask a core member to copy the DOI information you added to dev via the PR above to the master branch. The core member will uncomment the Zenodo-related `TODO` statement in the `Citation` section of the main `README.md` and add the DOI, as well as as updating the badge for the Zenodo DOI at the top of the main `README.md` e.- [nf-core/atacseq](https://github.com/nf-core/atacseq/blob/fa1e3f8993cd20e249b9df09d29c5498eff311d2/README.md).
-- [ ] (first release only) Ask a core member to change default branch from `dev` to `master`.
+- [ ] (**First release only**) Ask a core member to change default branch from `dev` to `master`.
 - [ ] (publication only) If a publication of the pipeline is being prepared, recommended [nf-core guidelines](/docs/guidelines/pipelines/recommendations/publication_credit) are followed.
+- [ ] Post on the [#bytesize](https://nfcore.slack.com/archives/C01LNCGJBMH) slack channel of your release, to begin arranging a 15m introductory bytesize talk about your shiny new pipeline!
```

Comment:
```suggestion
- [ ] Post on the [#bytesize_suggestion](https://nfcore.slack.com/archives/C01LNCGJBMH) slack channel of your release, to begin arranging a 15m introductory bytesize talk about your shiny new pipeline!
```

on mobile, can't figure out how to copy the channel link sorry. But needs updating